<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestEditDialog</class>
 <widget class="QDialog" name="TestEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑测试</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelCategory">
       <property name="text">
        <string>类别</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboCategory"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>名称</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelTarget">
       <property name="text">
        <string>目标ID</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editTarget"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelDetectable">
       <property name="text">
        <string>检测故障</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="editDetectable"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelIsolatable">
       <property name="text">
        <string>隔离故障</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="editIsolatable"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelCost">
       <property name="text">
        <string>预估费用</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="doubleCost">
       <property name="maximum">
        <double>100000000.000000000000000</double>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelDuration">
       <property name="text">
        <string>预估时间</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="doubleDuration">
       <property name="maximum">
        <double>100000000.000000000000000</double>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string>描述</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainDescription"/>
   </item>
   <item>
    <widget class="QLabel" name="labelMetrics">
     <property name="text">
      <string>指标 (JSON 对象)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainMetrics"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
